code/source/playfab/PlayFabClientInstanceApi.cpp

```diff
@@ -4716,17 +4718,19 @@ namespace PlayFab
         CallRequestContainer& container = static_cast<CallRequestContainer&>(*reqContainer);
         std::shared_ptr<PlayFabAuthenticationContext> context = container.GetContext();
 
-        LoginResult outResult;
+        LoginResultWithUser outResult;
```

Comment:
I can't find where this type is defined. I assume you're adding a user field to the class? Is that the only change?

Reply:
Oh, found it... sortof.. But not.
Client _Models.h is so big and the changes so extensive that it won't show the diff.
So that's fun.